pkg/config/config.go

```diff
@@ -242,18 +242,31 @@ func init() {
 	Datadog.SetDefault("apm_config.enabled", true)
 
 	// Logs Agent
+
+	// External Use: modify those paramteers to configure the logs-agent.
+	// enable the logs-agent:
 	BindEnvAndSetDefault("logs_enabled", false)
 	BindEnvAndSetDefault("log_enabled", false) // deprecated, use logs_enabled instead
-	BindEnvAndSetDefault("logset", "")
+	// collect all logs from all containers:
+	BindEnvAndSetDefault("logs_config.container_collect_all", false)
+	// collect all logs forwarded by TCP on a specific port:
+	BindEnvAndSetDefault("logs_config.tcp_forward_port", -1)
+	// add a socks5 proxy:
+	BindEnvAndSetDefault("logs_config.socks5_proxy_address", "")
+	// override the logs destination:
+	BindEnvAndSetDefault("logs_config.logs_dd_url", "") // must respect format `<HOST>:<PORT>`
+	BindEnvAndSetDefault("logs_config.logs_no_ssl", false)
```

Comment:
We've discussed about this already, let's keep those names, but I find it very confusing that these settings don't have `proxy` in their name.
Could we at least add comments regarding that?
`// override the logs destination:` seems wrong, I'd rather explicitly mention the proxy use case